web/lib/firebase/contracts.ts

```diff
@@ -121,6 +121,19 @@ export async function listAllContracts(): Promise<Contract[]> {
   return snapshot.docs.map((doc) => doc.data() as Contract)
 }
 
+export async function userHasCreatedContractToday(
+  userId: string
+): Promise<boolean> {
+  // uses utc time like the server
```

Comment:
Nice attention to detail!

Reply:
A very small style nit is to use capital letters at the start of a comment. At Google we even ended with a period, though I wouldn't enforce that.